core-galleon-pack/src/main/resources/feature_groups/core-management.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="core-management" xmlns="urn:jboss:galleon:feature-group:1.0">
```

Comment:
This feature group seems odd. It follows the pattern of a feature group per subsystem, which in other cases includes config details for the feature.  My concern with this one is "core management" is a much bigger concept than just this one subsystem.

General question: what is the benefit of feature groups like this one, discovery.xml and request-controller.xml?

Reply:
The idea was consistency of the building blocks (for subsystems at least) used to compose the configs. But I guess it didn't really help at the end. Ok, I'm gonna remove them and we find it useful, we we'll add the group back later.